site/source/docs/getting_started/bug_reports.rst

```diff
@@ -17,10 +17,13 @@ Please supply as much relevant information as possible, including:
 .. Tip:: Compile with ``EMCC_DEBUG=1`` and grab the **/tmp/emscripten_temp/emcc-\*** files (these include the bitcode and JavaScript in several stages). Note that the **emscripten_temp** directory should be emptied manually first, so it only contains new content!
 
 
-Fastcomp LLVM-Backend and Clang bugs
-====================================
+LLVM, wasm-ld, clang, Binaryen bugs
+===================================
 
-:ref:`LLVM Backend <LLVM-Backend>` bugs may instead be posted to the appropriate `Emscripten LLVM fork <https://github.com/emscripten-core/emscripten-fastcomp>`_ or `Emscripten Clang fork <https://github.com/emscripten-core/emscripten-fastcomp-clang>`_ **if you are certain** that the bug is specific to these repositories. If uncertain, bugs must be posted to the `main repository <https://github.com/emscripten-core/emscripten>`_.
+If uncertain, bugs can always be posted to the `main repository <https://github.com/emscripten-core/emscripten>`_. But if you are sure a bug is in an upstream project, you can file it there:
+
+- <LLVM bug tracker <https://bugs.llvm.org/>`_
```

Comment:
Should probably mention that we use the "libraries" product and the "Backend: WebAssembly" component.